### PR TITLE
Restore Dma

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,10 +19,11 @@ name = "timer-interrupt-rtfm"
...
 
 [dependencies]
-cortex-m = "0.5.8"
+cortex-m = ">=0.5.8,<0.7"
 nb = "0.1.1"
-cortex-m-rt = "0.6.7"
-stm32f1 = "0.6.0"
+cortex-m-rt = "0.6.8"
+#stm32f1 = "0.7.0"
+stm32f1 = {path = "/home/burrbull/stm32-rs/stm32f1"}
```
Comment (eldruin) [inline Cargo.toml]: A local path slipped in here

File: examples/adc.rs [hunk 2]
```diff
@@ -1,26 +1,23 @@
 #![deny(unsafe_code)]
+#![deny(warnings)]
```
Comment (eldruin) [inline examples/adc.rs]: I discovered not long ago that this is an anti-pattern. See [here](https://github.com/rust-unofficial/patterns/blob/master/anti_patterns/deny-warnings.md)

File: Cargo.toml (hunk 1 above)
Comment (burrbull) [reply Cargo.toml]: Sorry, this is an accident.

File: examples/adc.rs (hunk 2 above)
Comment (burrbull) [reply examples/adc.rs]: Then I will remove this from other examples.

**Comment by burrbull** [comment]
The next three days I will not be able to commit.
Therefore, if minor fixes are needed, please correct them yourself or wait for me.

**Pull request merged**